### PR TITLE
fix: nft transfer

### PR DESCRIPTION
**Pull request by dubloom**
Not sure at all if it useful or not,

I let you tell me

### CODE REVIEW

**Review by patriciobcs — CHANGES_REQUESTED** [review]
I run it and the it seems to be working conceptually but you didn't update the "ownerList" in the "SharedFund" contract, that list is used to display the current owners. Please update it.

File: src/SharedFund.sol [hunk 1]
```diff
@@ -48,7 +55,17 @@ contract SharedFund is ERC721 {
         require(_isApprovedOrOwner(_msgSender(), tokenId), "ERC721: caller is not token owner or approved");
 
         _transfer(from, to, tokenId);
+
+        //Update list and map
+        delete owners[from];
+        for (uint i = 0; i < ownersList.length; i++) {
```
Comment (enitrat) [inline src/SharedFund.sol:61]: instead of doing this you should just do 
`ownersList[I] = to`

once fixed you can merge

**Review by enitrat — APPROVED** [review]

File: src/SharedFund.sol (hunk 1 above)
Comment (dubloom) [reply src/SharedFund.sol:61]: Smart

Comment (patriciobcs) [reply src/SharedFund.sol:61]: Agree with @msaug

**Review by patriciobcs — APPROVED** [review]

**Pull request merged**